modules/vuetify/index.js

```diff
@@ -0,0 +1,16 @@
+const path = require('path')
+
+module.exports = function nuxtVuetify (options) {
+  // Add assets
+  this.options.head.link.push({
+    rel: 'stylesheet',
+    type: 'text/css',
+    href: '//fonts.googleapis.com/css?family=Material+Icons'
```

Comment:
Even then, I agree with @pi0 that it would be nice if you could disable this. Using code from 3rd party sources like googleapis should always be optional imho